### ISSUE TITLE
Anchu\Ftp\Ftp Object ( [connectionId:protected] => Resource id #286 )

### ISSUE THREAD

**Issue by bsor-dev** [harishanchu/Laravel-FTP#73]
What does this mean? My credentials are correct but I can't get my files while in ftp my files are there. However, in my other application that uses ftp anchu ftp too it working fine. The following previous day its working fine, just right now it throws me that kind of error

**Comment by pratikshad** [comment]
yes even I am getting the same response ...no idea what does it mean
